homeassistant/components/knx/light.py

```diff
@@ -142,6 +146,113 @@ def _device_unique_id(self) -> str:
             f"{self._device.blue.brightness.group_address}"
         )
 
+    @staticmethod
+    def _create_light(xknx: XKNX, config: ConfigType) -> XknxLight:
+        """Return a KNX Light device to be used within XKNX."""
+
+        def create_light_color(
```

Comment:
This function doesn't need to be nested here. We can move it to the module level.